src/purva/CodingExam1_18_04/Program1ReturnIndicesForTargetSum.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Program 1 :
+Given an array of integers nums and an integer target, return indices of the two numbers such that they add up to the target.
+
+Input: nums = [2,11,15,7,4], target = 9
+
+Output: [0,3]
+
+Output: Because nums[0] + nums[3] == 9, we return [0, 3].
+ */
+
+
+package purva.CodingExam1_18_04;
+
+public class Program1ReturnIndicesForTargetSum {
+	
+	int[] sumIndex(int[] num) {
+		int sum =0;
+		int count =0;
+		int[] returnIndicices = new int[2];
+		for(int index = 0; index<num.length;index++) {
+			for(int innerIndex=1;innerIndex<num.length;innerIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files